sdk/core/Azure.Core/tests/TestFramework/TestRecording.cs

```diff
@@ -264,22 +264,27 @@ public void DisableIdReuse()
 
         public DisableRecordingScope DisableRecording()
         {
-            return new DisableRecordingScope(this);
+            return new DisableRecordingScope(this, EntryRecordModel.DontRecord);
+        }
+
+        public DisableRecordingScope DisableRequestBodyRecording()
```

Comment:
I didn't have a test for the original `DisableRecording` feature to have as an example but it would be nice to add something for `DisableRequestBodyRecording`. SavingRecordingSanitizesValues test might be a good starting point.